recipes/test/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+export C_INCLUDE_PATH=${PREFIX}/include
+export LIBRARY_PATH=${PREFIX}/lib
+
+make CC=${CC} CFLAGS="${CFLAGS}" LDFLAGS="${LDFLAGS}" all
```

Comment:
```suggestion
make -j ${CPU_COUNT} CC=${CC} CFLAGS="${CFLAGS}" LDFLAGS="${LDFLAGS}" all
```
to make it parallel and thus use more CPU & RAM